adapters/repos/db/aggregator/text.go

```diff
@@ -86,6 +86,11 @@ func (a *textAggregator) insertOrdered(elem aggregation.TextOccurrence) {
 		if pair.Occurs > elem.Occurs {
 			continue
 		}
+		// if number of occurrences is the same,
+		// skip if string is after one in topPairs
+		if pair.Occurs == elem.Occurs && pair.Value < elem.Value {
+			continue
+		}
```

Comment:
Same as above, small unittest for this would be great (afaik there is not yet a test file for this)

Reply:
added

Reply:
thanks!